Source/Modules/php5.cxx

```diff
@@ -528,7 +530,14 @@ class PHP5 : public Language {
     Printf(s_entry, "/* Every non-class user visible function must have an entry here */\n");
     Printf(s_entry, "static zend_function_entry %s_functions[] = {\n", module);
 
+    /* Emit all of the code */
+    Language::top(n);
+
+    SwigPHP_emit_resource_registrations();
+
     /* start the init section */
+    String * s_init_old = s_init;
+    s_init = NewString("/* init section */\n");
```

Comment:
Hey, I've applied all the changes, do take a look again. And I've also added the Documentation and Example code as a separate commit.